accession-commons-mongodb/src/main/java/uk/ac/ebi/ampt2d/commons/accession/persistence/mongodb/repository/BasicMongoDbAccessionedCustomRepositoryImpl.java

```diff
@@ -97,6 +104,8 @@ private void setAuditCreatedDate(Iterable<DOCUMENT> documents) {
                 return Optional.of(matcher.group(1));
             }
         }
+        logger.error("Error parsing BulkWriteError in BulkOperationException code '" + error.getCode() + "' message '" +
+                error.getMessage() + "'");
```

Comment:
I think displaying this message makes things more confusing because if there is another error code it is not a parsing error. What about the following?

```java
if (11000 == error.getCode) {
    // ...
    if (matcher.find()) {
        return Optional.of(matcher.group(1));
    } else {
        logger.error("Error parsing BulkWriteError in BulkOperationException. Code: '" + error.getCode() 
                     + "'. Message: '" + error.getMessage() + "'");
    }
} else {
    logger.error("Unexpected BulkWriteError in BulkOperationException. Code: '" + error.getCode() 
                 + "'. Message: '" + error.getMessage() + "'");
}

return Optional.empty();
```